### PR TITLE
Add iPad support to the FluentUITester-iOS target

### PR DESCRIPTION
**Pull request by Saadnajmi**
### Platforms Impacted
- [x] iOS
- [ ] macOS
- [ ] win32
- [ ] windows
- [ ] android

### Description of changes

I noticed that the FluentUITester-ios test app didn't use the full iPad screen in simulator, and rendered as an iPhone app.
Click the checkbox in the test app target so it renders for iPad.

### Verification

Opened test app in iPad. It is now using the full screen.

Screen Shot 2020-07-09 at 12 47 03 PM

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/ios/src/FluentUITester.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -285,13 +285,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9F3A29F4244F657400A401D2 /* Debug.xcconfig */;
 			buildSettings = {
+				TARGETED_DEVICE_FAMILY = "1,2";
```
Comment (Saadnajmi) [inline apps/ios/src/FluentUITester.xcodeproj/project.pbxproj:288]: I'm gonna take a wild guess and say this should probably be moved to an xcconfig?

Comment (anandrajeswaran) [reply apps/ios/src/FluentUITester.xcodeproj/project.pbxproj:288]: Yes definitely!

**Comment by Saadnajmi** [comment]
CI Error. Can be reproduced by just running `yarn ios`. Great thing we got that running :D 
```
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 70. To debug build logs further, consider building your app with Xcode.app, by opening FluentUITester.xcworkspace. Run CLI with --verbose flag for more details.
Command line invocation:
    /Applications/Xcode_11.4.1.app/Contents/Developer/usr/bin/xcodebuild -workspace FluentUITester.xcworkspace -configuration Debug -scheme FluentUITester -destination id=94DB3E3C-59FF-47BD-A71C-11F19B451920


xcodebuild: error: Unable to find a destination matching the provided destination specifier:
                { id:94DB3E3C-59FF-47BD-A71C-11F19B451920 }

        Available destinations for the "FluentUITester" scheme:
                { platform:iOS Simulator, id:94434ADE-2F63-484F-B02D-453440EA42C6, OS:13.4.1, name:iPad (7th generation) }
                { platform:iOS Simulator, id:B6A6D4F5-7022-459F-BFBF-26AD3CF8BDFD, OS:13.4.1, name:iPad Air (3rd generation) }
                { platform:iOS Simulator, id:7165ACDB-D8C5-4300-A97E-5199A28DF407, OS:13.4.1, name:iPad Pro (9.7-inch) }
                { platform:iOS Simulator, id:A8E6CED4-C0A8-484F-A86A-B596DABC4430, OS:13.4.1, name:iPad Pro (11-inch) (2nd generation) }
                { platform:iOS Simulator, id:35A7EF3C-FB49-4893-BECE-1DAAD1474BE5, OS:13.4.1, name:iPad Pro (12.9-inch) (4th generation) }

        Ineligible destinations for the "FluentUITester" scheme:
                { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Generic iOS Device }
                { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Generic iOS Simulator Device }
```

**Comment by Saadnajmi** [comment]
Thanks to #302 , we no longer need this

**Pull request closed without merging**